blackbox/docs/configuration.txt

```diff
@@ -990,51 +990,40 @@ Discovery on Microsoft Azure
 Crate has built-in support for discovery via the Azure Virtual Machine API.
 To enable Azure discovery set the ``discovery.type`` setting to ``azure``.
 
-**cloud.azure.management.keystore.path**
+**cloud.azure.management.resourcegroup.name**
   | *Runtime:*  ``no``
 
-  Path to the keystore.
-
-**cloud.azure.management.keystore.type**
-  | *Runtime:*  ``no``
-  | *Default:* ``pkcs12``
-  | *Allowed Values:* ``pkcs12 | jceks | jks``
-
-  The type of the keystore.
-
-**cloud.azure.management.keystore.password**
-  | *Runtime:*  ``no``
-
-  The password of the keystore.
+    the azure resource group name
 
 **cloud.azure.management.subscription.id**
   | *Runtime:*  ``no``
 
-  The Id of your Azure subscription, in the format of a UUID
-  ``xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx``.
+    the azure subscription id
 
-**cloud.azure.management.cloud.service.name**
+**cloud.azure.management.tenant.id**
   | *Runtime:*  ``no``
 
-**discovery.azure.host.type**
-  | *Runtime:*  ``no``
-  | *Default:* ``private_ip``
-  | *Allowed Values:* ``public_ip | private_ip``
-
-  Defines via which host type to communicate with other instances.
+    the azure tenant id
 
-**discovery.azure.endpoint.name**
+**cloud.azure.management.app.id**
   | *Runtime:*  ``no``
 
-**discovery.azure.deployment.name**
-  | *Runtime:*  ``no``
+    the azure active directory application id
 
-**discovery.azure.deployment.slot**
+**cloud.azure.management.app.secret**
   | *Runtime:*  ``no``
-  | *Default:* ``production``
-  | *Allowed Values:* ``staging | production``
 
+    the password for the azure active directory application
+
+**discovery.azure.method**
+  | *Runtime:* ``no``
+  | *Default:* ``vnet``
+  | *Allowed Values:* ``vnet | subnet``
 
+     Discovery method ``vnet`` means that all vms
```

Comment:
suggestion:

Discovery method ``vnet`` will discover all VMs in the same virtual network.
The use of method ``subnet`` will add all virtual machines in the same subnet to the cluster.